pkg/api/norman/customization/project/project_actions.go

```diff
@@ -273,23 +277,28 @@ func (h *Handler) setPodSecurityPolicyTemplate(actionName string, action *types.
 	providedPSP := input[client.PodSecurityPolicyTemplateProjectBindingFieldPodSecurityPolicyTemplateName]
 
 	if providedPSP != nil {
-		// cannot use cluster manager to get cluster name as subcontext is nil
+		// cannot use clusterv3 manager to get clusterv3 name as subcontext is nil
 		idParts := strings.Split(request.ID, ":")
 		if len(idParts) != 2 {
 			return httperror.NewAPIError(httperror.InvalidBodyContent,
-				fmt.Sprintf("cannot parse cluster from ID [%s]", request.ID))
+				fmt.Sprintf("cannot parse clusterv3 from ID [%s]", request.ID))
 		}
 
 		clusterName := idParts[0]
-		cluster, err := h.ClusterLister.Get("", clusterName)
+		clusterv3, err := h.ClusterLister.Get("", clusterName)
+		if err != nil {
+			return fmt.Errorf("error retrieving clusterv3 [%s]: %v", clusterName, err)
+		}
+
+		clusterv1, err := h.ClusterListerEx.Get("fleet-default", clusterv3.Spec.DisplayName, metav1.GetOptions{})
 		if err != nil {
-			return fmt.Errorf("error retrieving cluster [%s]: %v", clusterName, err)
+			return fmt.Errorf("error retrieving clusterv1 [%s]: %w", clusterv3.Spec.DisplayName, err)
```

Comment:
We should try to refer to `v1` cluster as the management cluster and the `v3` cluster as the provisioning cluster so it's clearer to other's who don't know the differentiation between the two versions

Reply:
names updated